packages/block-editor/src/components/font-sizes/fluid-utils.js

```diff
@@ -68,15 +70,54 @@ export function getComputedFluidTypographyValue( {
 		// If no minimumFontSize is provided, derive using min scale factor.
 		if ( ! minimumFontSize ) {
 			minimumFontSize =
-				fontSizeParsed.value * minimumFontSizeFactor +
-				fontSizeParsed.unit;
+				roundToPrecision(
+					fontSizeParsed.value * minimumFontSizeFactor,
+					3
+				) + fontSizeParsed.unit;
+		}
+
+		// Parses the minimum font size limit, so we can perform checks using it.
+		const minimumFontSizeLimitParsed = getTypographyValueAndUnit(
+			minimumFontSizeLimit,
+			{
+				coerceTo: fontSizeParsed.unit,
+			}
+		);
+
+		/*
+		 * Checks if a user-defined font size is lower than $minimum_font_size_limit. If so than it should become the minimum font size.
```

Comment:
Same typo as above 😄

```suggestion
		 * Checks if a user-defined font size is lower than $minimum_font_size_limit. If so then it should become the minimum font size.
```